dev-python/pep517/pep517-0.13.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022-2023 Gentoo Authors
```

Comment:
We already have dev-python/pyproject-hooks in tree - is it the same thing or different?

Reply:
It's the predecessor of this project. TBH, I don't even think it's used in pipenv. I'll have to check with upstream.
It could be that the ebuild isn't even needed.

Reply:
Patch submitted upstream to remove pep517. Thanks for that.